src/sharepoint/account/ACCOUNT.html

```diff
@@ -124,12 +124,19 @@
         <div class="col-md-3 mt-md-5 mt-lg-0">
             <button class="btn btn-block btn-default text-wrap" title="{{ 'sharepoint_accounts_action_reset_admin_tooltip' | translate }}"
                     data-translate="sharepoint_accounts_action_reset_admin"
-                    data-ng-click="accountsCtrl.resetAdminRights()">
+                    data-ng-click="$ctrl.resetAdminRights()">
             </button>
-            <button class="btn btn-block btn-default text-wrap" title="{{ 'sharepoint_exchange_ACTION_order_accounts_button' | translate }}"
+            <button class="btn btn-block btn-default text-wrap" type="button"
+                    title="{{ 'sharepoint_exchange_ACTION_order_accounts_button' | translate }}"
+                    data-translate="sharepoint_exchange_ACTION_order_accounts_button"
+                    data-ng-click="setAction('account/add-legacy/sharepoint-account-add-legacy', {})"
+                    data-ng-if="$ctrl.isStandAlone && $ctrl.usesAgora !== null && !$ctrl.usesAgora">
```

Comment:
```suggestion
                    data-ng-if="$ctrl.isStandAlone && !$ctrl.usesAgora">
```

should be enough

Reply:
`usesAgora === null` means it's still loading, while a boolean value (may it be `true` or `false`) means we got the answer. Maybe I should make it more obvious with a specific "loading" variable?